modules/server_processes_manager/test/TestPlan.md

```diff
@@ -23,3 +23,4 @@ This test plan should be executed by someone with decent knowledge of the MRI up
    When the process finishes (when the exit code file contains something), verify that all the process files
    have been deleted from `/tmp` if and only if the exit code is zero.
 
+- Note: If the page doesn't load add the following path `/opt/Loris-MRI/bin/mri/` to the LORIS-MRI code path in configuration module.
```

Comment:
I think the path could vary project by project, so we just need to specify that the LORIS-MRI code path should be set without specifying what exactly it would be (also applies to the above file)